source/docs/advanced/features/disable-hyration.md

```diff
@@ -0,0 +1,33 @@
+---
+id: disable-hyration
+title: Disable Hydration
+---
+
+# What is Hydration?
```

Comment:
An introduction is missing for context of this page. As per !408, it should be splitted into a `description:` and introduction paragraph.

If that feels like too much to do, then this page could also be reworked as a paragraph in the SSR page.

Reply:
I added a discription in similar manner to !408 the introduction remains the same. is that satisfactory?